scripts/automation/watcher.py

```
@@ -0,0 +1,148 @@
+
+
+def scan_once(client, args) -> int:
+    """1 周スキャンして、検出された全マニュアルを処理。処理件数を返す。"""
+    pending = find_pending_rows(client)
+    if not pending:
+        return 0
+    print(f"処理対象: {len(pending)} 件")
+    for name, mode in pending:
+        try:
+            run_process_one(name, mode, args)
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)